Add pollenflug extension

h16nning opened the pull request:
## Description

Adds an extensions showing current pollen count provided by DWD's api (German Weather Services).

One can choose between the different regions, pin those pollen types they are allergic too and cycle through data for today, tomorrow and the day after.

## Screencast
![pollenflug-1](https://github.com/user-attachments/assets/8a2eb218-85f6-42f4-931e-a410b068ac5c)
![pollenflug-2](https://github.com/user-attachments/assets/a4ad4bb5-bb8d-46ad-b750-96f91baae1ee)
![pollenflug-3](https://github.com/user-attachments/assets/840a20d0-460d-4966-88a5-2dbb566f3247)


## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

h16nning commented on extensions/pollenflug/src/locations.ts:
```diff
@@ -0,0 +1,176 @@
+  },
+  {
+    partregion_id: 12,
+    partregion_name: "Geest,Schleswig-Holstein und Hamburg",
```
This is data as it is coming from api

h16nning commented on extensions/pollenflug/src/locations.ts:
```diff
@@ -0,0 +1,176 @@
+    region_name: "Schleswig-Holstein und Hamburg",
+  },
+  {
+    partregion_id: -1,
+    partregion_name: "",
```
This is data as it is coming from api

pernielsentikaer reviewed (commented):
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.

h16nning commented:
Hello @pernielsentikaer! I adressed the changes by removing my custom German translations for pollen severity. I also introduced English translations for data provided by the api:  for the pollen names (Birche → Birch, Ambrosia → Ragweed, …) and for the region names (Nordrhein-Westfalen → Northrhine-Westphalia etc., if there were translations for German geographic regions)

I renamed the command from `show-pollenflug` to `show-pollen-count`.

I also renamed the extension from `pollenflug` to `pollen-count` in the package.json. However, I wasn't able to figure out to rename the root folder 🤔 so that one is still `pollenflug`. Can we change that?

h16nning commented:
Hello, are there any further requirements to merge this? I’d be more than happy to implement them and see this extension in the store😊

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
